org.alloytools.alloy.api/Alloy.md

```diff
@@ -0,0 +1,375 @@
+---
```

Comment:
This makes it render as some kind of table.  Is that intentional?  Why not just 
```
# Alloy API Overview
```

Reply:
This is the YAML header we need for making this a web page on the web site. In Alloy, it is the marker that sees it is not Alloy syntax but markdown with embedded Alloy